pyproj/_crs.pyx

```diff
@@ -470,7 +470,7 @@ cdef class Ellipsoid(Base):
             PJ_CATEGORY_ELLIPSOID,
         )
         if ellipsoid_pj == NULL:
-            return None
+            raise CRSError("Invalid authority")
```

Comment:
```suggestion
            raise CRSError("Invalid authority or code ({0}, {1})".format(auth_name, code))
```

And something similar for the others as well, that gives a bit more informative error message.

Reply:
Sounds like a good idea.